### DISCUSSION TITLE
❓ Questions / Help / Support - where do i download silero_lang_detector for onnx

### DISCUSSION THREAD

**Discussion by Goddard** [snakers4/silero-vad#328] [Q&A] [answered]
I am looking to get the model working with a nodejs application and I am curious where I can just download the model?  Is it apart of the vad model?  The other models all have their urls mapped out but the lang detector is always just being loaded by torch hub.

**Comment by snakers4** [comment, marked as answer]
Hi,

The lang detector is deprecated now.
But you can find the url in the hubconf file.

**Reply by weimeng23** [reply]
Hi  @snakers4 
models.silero.ai 404 Not Found, can't download anymore.

**Reply by snakers4** [reply]
We faced a large amount of DDOS traffic repeteadly downloading these model `silero_lang_detector`, i.e. _**10-20 download requests PER SECOND**_.
So we deprecated the models fully - now the endpoints creating these models raise an error and models are not available anymore.

**Reply by weimeng23** [reply]
how about uploading models to huggingface ？

**Reply by snakers4** [reply]
No.

**Reply by weimeng23** [reply]
All right 🤣
